.gitignore

```diff
@@ -87,3 +87,4 @@ ENV/
 
 # Rope project settings
 .ropeproject
+.pytest
```

Comment:
I see this mention to pytest. Could we write a unit test for the address problem?